web/src/components/nomination/NominateForm.vue

```diff
@@ -87,6 +87,13 @@
       @touch-twitter="delayTouch($v.form.urls.twitter)"
       @touch-website="delayTouch($v.form.urls.website)"
     />
+    <v-checkbox
+      v-model="isSelfNomination"
+      class="mt-0"
+      label="I am nominating myself"
```

Comment:
Oh, just noticed this. We should be using vue-i18n values for all text, rather than raw strings. Can you please add the new key `selfNomination` to en.json and then reference it with the `$t('selfNomination')` here?

Reply:
yeah I'll update this to use i18n 👍

Reply:
I added it here:
https://github.com/WWCodeTokyo/speak-her-db/pull/167/commits/7e1b4c027d1154c63592e6859be859c441147289

I added the japanese translation as well